daprdocs/content/en/reference/components-reference/supported-bindings/commercetools.md

```diff
@@ -0,0 +1,71 @@
+---
+type: docs
+title: "commercetools GraphQL binding spec"
+linkTitle: "commercetools GraphQL"
+description: "Detailed documentation on the commercetools GraphQL binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/commercetools/"
+---
+
+## Component format
+
+To setup commercetools GraphQL binding create a component of type `bindings.commercetools`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: bindings.commercetools
+  version: v1
+  metadata:
+  - name: region # required.
+    value: region
+  - name: provider # required.
+    value: provider (gcp/aws)
+  - name: projectKey # required.
+    value: project-key
+  - name: clientID # required.
+    value: *****************
+  - name: clientSecret # required.
+    value: *****************
+  - name: scopes # required.
+    value: scopes
+
+```
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Binding support |  Details | Example |
+|--------------------|:--------:|------------|-----|---------|
+| Region | Y | Output | The region of the commercetools project | `"europe-west1"` |
```

Comment:
These fields should all start with a lowercase character to match the code PR:

https://github.com/dapr/components-contrib/pull/1652/files

Reply:
@berndverst done